app/javascript/components/util/ConfirmDialog.js

```diff
@@ -17,6 +17,7 @@ class Confirmation extends React.Component {
   render() {
     const {
       okLabel = 'OK',
+      okVariant = 'primary',
```

Comment:
should we define `cancelVariant` as well for consistency?

Reply:
I don't think so - the cancel button should always look the same (as it does across the app)